collector/setting.go

```diff
@@ -0,0 +1,96 @@
+package collector
+
+import (
+	rancher "github.com/rancher/types/client/management/v3"
+	log "github.com/sirupsen/logrus"
+)
+
+func GetSetting(client *rancher.Client, key string) (string, bool) {
```

Comment:
Can you add some comments to func's in this file. I think most of other collector methods are self-explanatory, but all these settings ones are hard to tell apart.

Reply:
Cleaned up `GetSetting ` function due to not needed.